README.md

```diff
@@ -1,2 +1,20 @@
 # concrete-ml-fuzzer
 A differential fuzzer to compare [concrete-ml](https://docs.zama.ai/concrete-ml) against [scikit-learn](https://scikit-learn.org)
+
+## Dependencies
+- scikit-learn
+- concrete-ml
+- atheris
+
+These can be installed with:
+```sh
+pip install concrete-ml atheris
+```
+Installing concrete-ml with pip will install `scikit-learn` 
+
+## Run the fuzzer
+
+```sh
+cd src
+python3 logistic_regression.py
```

Comment:
As this is the general README put something like p`ython3 <fuzzer_file.py>`

Reply:
Resolved: https://github.com/lambdaclass/concrete-ml-fuzzer/pull/3/commits/b71e74af30a017ba25510b809ec6e542d5d14f81